pyproject.toml

```diff
@@ -125,9 +126,12 @@ wheels = [
 
 [tool.cibuildwheel]
 skip = [
-    "cp314t-*",  # Disabled due to instability concerns.
     "cp310-win_arm64", # Skipped due to unavailability of Python binary and NumPy for Win-ARM64
 ]
+enable = [
+    "cpython-freethreading",  # Build cp313t wheels
+]
```

Comment:
Let's delete this, per https://github.com/h5py/h5py/pull/2702#issuecomment-3330232792.